### PR TITLE
Reduce prometheus memory footprint by reduced labelling and defensive querying

### PR DESCRIPTION
**Pull request by hisarbalik**
#7036

### CODE REVIEW

File: resources/event-bus/charts/event-publish-service/templates/service-monitor.yaml [hunk 1]
```diff
@@ -16,5 +16,9 @@ spec:
 {{- if .Values.monitoring.scrapeInterval }}
     interval: {{ .Values.monitoring.scrapeInterval }}
 {{- end }}
+    metricRelabelings:
+    - sourceLabels: [ __name__ ]
+      regex: ^(go_goroutines|go_memstats_alloc_bytes|go_memstats_heap_alloc_bytes|go_memstats_heap_inuse_bytes|go_memstats_heap_sys_bytes|go_memstats_stack_inuse_bytes|process_cpu_seconds_total|process_max_fds|process_open_fds|process_resident_memory_bytes|process_start_time_seconds|process_virtual_memory_bytes)$
```
Comment (sayanh) [inline resources/event-bus/charts/event-publish-service/templates/service-monitor.yaml]: `kyma-event-bus` dashboard does not have any data. It uses `total_published_messages` metrics. Do we need to include this here?

**Comment by jasiu001** [comment]
Some metrics for HelmBroker are missing, GC duration shouldn't be empty. 
Below is a comparison with the `weekly` cluster:

Screen Shot 2020-02-27 at 13 40 57
Screen Shot 2020-02-27 at 13 41 12

The same situation is for BindingUsageComponent. (as an above comparison with `weekly`)

Screen Shot 2020-02-27 at 14 16 57
Screen Shot 2020-02-27 at 14 18 42

**Comment by hisarbalik** [comment]
> What `my-kyma.yaml` is used for?

File removed, was mistakenly added

**Comment by hisarbalik** [comment]
> Currently, metrics for Application Registry and Connector Service do not work here and on the master branch. There is already [an issue for it](https://github.com/kyma-project/kyma/issues/7478)
> I'll approve this PR from the Application Connector side not to block it. Potential oversights should be fixed as a follow-up

Be aware, after metric re-labeling some labels may be not more available with old names like instead of **pod_name** will be **pod** etc.

**Comment by sjanota** [comment]
There is no memory, network and CPU information for Kyma / WebAPPs.
Screen Shot 2020-03-05 at 15 01 31
They are available on weekly.

**Comment by hisarbalik** [comment]
> There is no memory, network and CPU information for Kyma / WebAPPs.
> Screen Shot 2020-03-05 at 15 01 31
> They are available on weekly.

dashboard updated, please check again

**Pull request merged**